### DISCUSSION TITLE
Docker-comopse.yml

### DISCUSSION THREAD

**Discussion by WaiserHai** [AzuraCast/AzuraCast#5984] [Q&A]
Hi folks, does anyone have a working Docker-comopse.yml file for me? 
I need one without ssl certificate because i control it via npm. I would be very grateful for that.

Thanks

**Comment by BusterNeece** [comment]
See our documentation for how to create an override file to provide your own SSL certificate if you like (if you don't provide one, by default we will just use a self-signed certificate, which should also work with most setups): https://docs.azuracast.com/en/administration/ssl-and-lets-encrypt#using-a-custom-certificate
